test/functionalTest/testHarness.sh

```diff
@@ -139,6 +139,7 @@ function usage()
   echo "$empty [--noCache (force broker to be started with the option --noCache)]"
   echo "$empty [--cache (force broker to be started without the option --noCache)]"
   echo "$empty [--noThreadpool (do not use a threadpool, unless specified by a test case. If not set, a thread pool of 200:20 is used by default in test cases which do not set notificationMode options)]"
+  echo "$empty [--xbroker (use an external broker)]"
```

Comment:
"(use an external broker)" vs "(use external brokers)" ?

We have some test cases that use several broker instances (the ones for CPrs cases mainly). If `--xbroker` disables all them (not sure) then I think the second wording is more precise.

Reply:
Fixed in e0bb95f